GetOldTweets3/manager/ConcurrentTweetManager.py

```diff
@@ -0,0 +1,78 @@
+# Author: Giulio Neusch-Frediani - www.github.com/giulionf
+from datetime import datetime
+from threading import Thread
+from queue import Queue
+from queue import Empty
+import copy
+
+from GetOldTweets3.manager.TweetManager import TweetManager
+
+DATE_FORMAT = "%Y-%m-%d"
+FIRST_TWEET_DATE = datetime.strptime("2006-03-21", DATE_FORMAT)
+
+
+class ConcurrentTweetManager:
+
+    @staticmethod
+    def getTweets(tweetCriteria, receiveBuffer=None, bufferLength=100, proxy=None, debug=False, worker_count=1, ordered=True):
+        # Init the queues
+        time_spans = Queue()
+        tweets = []
+        workers = []
+
+        # Split the date in smaller parts
+        since_date = datetime.strptime(tweetCriteria.since, DATE_FORMAT) if tweetCriteria.since else FIRST_TWEET_DATE
+        until_date = datetime.strptime(tweetCriteria.until, DATE_FORMAT) if tweetCriteria.until else datetime.now()
+        date_diff_per_worker = (until_date - since_date) / worker_count
```

Comment:
Need to check that `date_diff_per_worker` is 1 day at least

Reply:
True! Good catch!